frontends/ol-components/src/components/LearningResourceCard/LearningResourceListCard.tsx

```diff
@@ -121,7 +133,10 @@ const Info = ({ resource }: { resource: LearningResource }) => {
         <Certificate>
           <RiAwardFill />
           <CertificateText>
-            {resource.certification_type?.name || "Certificate"}
+            <MobileOnly>Certificate</MobileOnly>
+            <DesktopOnly>
+              {resource.certification_type?.name || "Certificate"}
```

Comment:
Related to above comment... Surprisingly, `certificate_type` is actually a required property. Things with `certification: false` (Free courses and videos and podcasts, ... ) have `certification_type: { code: "none", name: "No Certificate" }`).

No objection to the fallback... It is odd that certificate_type is required.

I suspect we added the fallback because tests were failing because the factory doesn't have it.